### ISSUE TITLE
New stdout/stderr stuff fails if either stdout or stderr is null/undefined

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/gcc-explorer#163]
https://sentry.io/matt-godbolt/compiler-explorer/issues/172474954/

```
TypeError: Unable to get property 'concat' of undefined or null reference
  at self (compiler.js:294)
  at list (compiler.js:197)
  at c (ext/jquery/dist/jquery.js:3305)
  at deferred (ext/jquery/dist/jquery.js:3438)
  at complete (ext/jquery/dist/jquery.js:9242)
...
(2 additional frame(s) were not displayed)
```

**Comment by mattgodbolt** [comment]
https://sentry.io/matt-godbolt/compiler-explorer/issues/172324737/

**Comment by mattgodbolt** [comment]
https://sentry.io/matt-godbolt/compiler-explorer/issues/175351038/

**Comment by mattgodbolt** [comment]
Still happening...fix is in 376e472a6c2c9a81a57fed496f20b9920fe7853e but not pushed to prod yet...

**Comment by mattgodbolt** [comment]
Seems fixed at last
